src/components/CowPenContextMenu/CowPenContextMenu.js

```diff
@@ -21,7 +21,7 @@ import {
   nullArray,
 } from '../../utils'
 import { PURCHASEABLE_COW_PENS } from '../../constants'
-import { cowFeed, huggingMachine } from '../../data/items'
+import shopInventory from '../../data/shop-inventory-cow'
```

Comment:
What do you think about renaming `shopInventory` to something like `cowShopInventory` to match `utils.js`?